jep/README.md

```diff
@@ -0,0 +1,142 @@
+---
+title: React.js at Jupyter
+authors: Eric Charles (@echarles)
+issue-number: <pre-proposal-issue-number>
+pr-number: <proposal-pull-request-number>
+date-started: 2022-02-08
+---
+
+> JEP for Jupyter UI Components
+
+This is a draft content towards a JEP for `Jupyter UI Components` based on the official [template](https://github.com/jupyter/enhancement-proposals/blob/master/jupyter-enhancement-proposal-guidelines/JEP-TEMPLATE.md).
+
+Once more preliminary feedback gathered, it will be submitted as a JEP (Jupyter Enhancement Proposal). We welcome contributors and reviewers in this early inception phase. More information on the JEP process can be read on:
+
+- [Guidelines](https://jupyter.org/enhancement-proposals/jupyter-enhancement-proposal-guidelines/jupyter-enhancement-proposal-guidelines.html)
+- [Repo](https://github.com/jupyter/enhancement-proposals)
+- [Template](https://github.com/jupyter/enhancement-proposals/blob/master/jupyter-enhancement-proposal-guidelines/JEP-TEMPLATE.md)
+
+# Summary
+
+This JEP proposes the adoption of well defined `UI Components` accross Jupyter projects that need to build user interfaces. While being primarily focussed on [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and [React.js](https://reactjs.org) technology, the creation of components based on other technologies like [Vue.js](https://vuejs.org), [Svelte](https://svelte.dev)... will be possible.
+
+The deliverable will be the creation of a `ui-components` repository providing accessible and themed widgets fully compatible with React.js, without blocking usage by other popular frontend technologies.
+
+On top of those components, extended functionalities on `reactivity` and `collaboration` are also discussed here. But they are out-of-scope of the deliverable. It makes sense to already discuss those last aspects in relationship with [Jupyter Widgets](https://github.com/jupyter-widgets) and the current JupyterLab Realtime collaboration features.
+
+Accessibility criteria will drive the creation of those components.
```

Comment:
Do you know what criteria? If so, it may be worth noting to give some credit. 

I'm guessing you are thinking of [WCAG](https://www.w3.org/TR/WCAG21/)?

Reply:
Indeed the components follow WCAG 2.1.

Reply:
Current text takes this into account.thx.